### ISSUE TITLE
How to activate ApplicationCharge..?

### ISSUE THREAD

**Issue by salman0802** [phpclassic/php-shopify#75]
Hello,
i'm getting error like "No method activate in application change" this if i use following code $shopifySDK->ApplicationCharge($charge_id)->activate();

Please help to solve this issue.

**Comment by kwasnicki** [comment]
Did you figure this out?
I have the same question except with the RecurringApplicationCharge
$shopify->RecurringApplicationCharge($charge_id)->activate();

Thanks
ken

**Comment by kwasnicki** [comment]
Update:  I just got a successful response by including the same array of data fields in activate() as I did in the original post().  ie:
```
$response = $shopify->RecurringApplicationCharge($charge_id)->activate(
      [
        "name" => "app/plan name",
        "price" => 15,
        "return_url" => "{return url}",
        "test" => true
      ]
    );
```

**Comment by tareqtms** [comment]
Fixed with #76
